altos-rust/altos-core/src/init.rs

```diff
@@ -17,7 +17,7 @@
 
 // FIXME: Try to see if there's a better way to handle this for testing
 // We do this cfg for testing purposes, this allows doctests to run without any compilation errors.
-#![cfg(all(not(test), not(feature="test"), feature="bump_allocator"))]
```

Comment:
This should keep the `feature="bump_allocator"` attribute so the bump allocator can be used still